CRM/Utils/System/Backdrop.php

```diff
@@ -1186,4 +1186,14 @@ public function theme(&$content, $print = FALSE, $maintenance = FALSE) {
     }
   }
 
+  /**
+   * @inheritdoc
+   */
+  public function ipAddress():?string {
+    // Backdrop function handles the server being behind a proxy securely. We
+    // still have legacy ipn methods that reach this point without bootstrapping
```

Comment:
This comment seems to contradict the code? You might as well include that function_exists to avoid unexpected calamities?